olm/quickstarts/rhosak-openshift-quarkus-bind-quickstart.yaml

```diff
@@ -3,223 +3,230 @@ kind: ConsoleQuickStart
+        - Your Kafka instance in [Streams for Apache Kafka](https://cloud.redhat.com/beta/application-services/streams/) is in the **Ready** state.
+        - You've created the `prices` topic in your Kafka instance.
 
-        1. Make sure that your OpenShift **Project**, which you can see at the top of the **Add** window, is set to `{username}-dev` (where *{username}* is your username in the DevSandbox OpenShift environment).
+        #### Procedure
 
-        1. Click on the **Container Image** card.
+        1. If you used an image provided by Streams for Apache Kafka to install CLI tools in your OpenShift project:
 
-        1. In the **Image name from external registry** field, enter: `quay.io/rhosak/rhoas-tools`.
+            i. Ensure that you're logged in to the OpenShift web console as the same user who deployed the Quarkus application in the previous task.
 
-        1. In the *Runtime icon* field, select `openshift`.
+            ii. In your project, click the icon for the tools application.
+                A sidebar opens, with the **Resources** tab displayed. Under **Pods**, you should see a single Pod, corresponding to the tools application.
 
-        1. Uncheck the **Route** box and leave all other fields set to their default values.
-        You don't need a route for this application, so under the **Advanced Options** you can uncheck the **create a route to the Application** checkbox. 
-        Click the **Create** button. This will create a new OpenShift **Deployment** for the tools image.
+            iii. Click the link to the Pod.
+                 The **Pod details** page opens.
 
-        Pulling the container image and creating the container will take a couple of seconds.
+            iv. Click the **Terminal** tab.
+                A command-line window opens inside the Pod.
 
-        1. You will see the deployment of the tools image in the [Topology]{{highlight qs-nav-add}} screen. The icon of the application should have a blue circle around it, indicating that the application has been deployed successfully.
+        1. If you deployed CLI tools locally on your machine:
 
-        1. Click on the icon of the tools application. This will open a panel on the righ-hand-side of your screen.
-        Click on the **Resources** tab. You will see the **Pods** of your Deployment. Currently we only have a single pod.
+            i. Open a command-line window.
 
-        1. Click on the link to the pod. This opens the details page of the pod.
+            ii. Log in to OpenShift as the same user who deployed the Quarkus application in the previous task.
 
-        1. Open the **Terminal** tab. This opens a terminal inside the pod. To check whether you can access the required tooling, execute the command `oc` in your terminal.
-        This should print the OpenShift Client CLI help text.
+           ```
+           $ oc login -u <user> -p <password> --server=<host:port>
```

Comment:
Why don't we instruct the user to login to OpenShift using the "Copy login command" option they get when clicking on their username in the upper right corner of the OpenShift Console? That gives a command that logs them in using their API-key, e.g.: "oc login --token=sha256~0WJOGcha7EOAkCmJpSSb6pyo2EawSUwZJKDEw3c-Ox4 --server=https://api.crc.testing:6443"

I find that especially convenient when I login to an environment that uses SSO, and/or an environment of which the URL of the OpenShift API is not obvious (e.g. codeready containers).